account_banking_payment_export/models/account_payment.py

```diff
@@ -35,6 +35,12 @@ class PaymentOrder(models.Model):
         readonly=True, states={'draft': [('readonly', False)]})
     mode_type = fields.Many2one('payment.mode.type', related='mode.type',
                                 string='Payment Type')
+    total = fields.Float(compute='_total', store=True)
+
+    @api.depends('line_ids', 'line_ids.amount')
+    @api.one
+    def _total(self):
```

Comment:
Minor thing: please call it `_compute_total` to follow conventions about compute methods names